pkg/contracts/src/CVStrategy.sol

```diff
@@ -403,7 +403,14 @@ contract CVStrategy is BaseStrategy, IPointStrategy, ERC165 {
     }
 
     function decreasePowerQuadratic(address _member, uint256 _amountToUnstake) internal view returns (uint256) {
-        uint256 newTotalPoints = (registryCommunity.getMemberStakedAmount(_member) - _amountToUnstake);
+        uint256 decimal = 18;
+        try ERC20(address(registryCommunity.gardenToken())).decimals() returns (uint8 _decimal) {
+            decimal = uint256(_decimal);
+        } catch {
+            console.log("Error getting decimal");
+        }
```

Comment:
should we revert here? should we have that in the initialization to avoid try catch here?